src/components/input_fields/__tests__/input_field_validation.test.tsx

```diff
@@ -6,35 +6,35 @@ import { shallow } from 'enzyme';
 import InputFieldValidation from '../input_field_validation';
 
 describe('The InputFieldValidation component', () => {
-    let props;
-    let wrapper;
+    let props
```

Comment:
This file is missing all ; now as only one.

Reply:
pycharm :cry: